detection-rules/link_published_google_doc.yml

```
@@ -0,0 +1,90 @@
+name: "Link: Published Google Drive Document With Suspicious Link"
```

this would follow the names of similar rules, also changes from "Google Drive" to "Google Docs" which better reflects the rule. 

```suggestion
name: "Link: Multistage Landing - Published Google Doc"
```